src/test/java/domain/LottoMachineTest.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatCode;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LottoMachineTest {
+
```

Comment:
```suggestion
class LottoMachineTest {

```

junit5 부터는 테스트가 public이 아니여도 됩니다!